src/backend/lib/redis.js

```diff
@@ -24,7 +31,7 @@ if (useMockRedis && typeof MockRedis.prototype.info !== 'function') {
 
 module.exports = {
   // Redis URL to use if using the constructor
-  redisUrl,
+  redisOptions,
```

Comment:
Instead of this, let's only expose a function that can create a Redis client connection using the host, password, port info from within this module, but not leak that out:

```js
function createRedisClient() {
  return new Redis(port, host, { password });
}
```